packages/moonstone/Scrollable/Scrollable.js

```diff
@@ -232,6 +232,12 @@ class ScrollableBase extends Component {
 		configureSpotlightContainer(nextProps);
 	}
 
+	componentDidUpdate () {
+		if (this.uiRef.scrollToInfo === null && this.childRef.nodeIndexToBeFocused == null) {
+			this.calculateAndScrollTo();
+		}
+	}
```

Comment:
With the code above, ENYO-5480 issue could be reproduced. We don't recommend to add any code in `componentDidUpdate` if possible.